introns.py

```diff
@@ -149,30 +166,39 @@ def create_introns(self):
         if len(self.exons) < 1:
             raise ValueError('No exons specified.')
             return
-        else:
+        elif len(self.exons) > 1:
```

Comment:
chyba nie uwzględniamy sytuacji kiedy len(self.exons)==1

Reply:
to jest specjalnie, jeśli len(self.exons) == 1, to nie będziemy szukać intronów